driver/src/main/java/org/kaazing/k3po/driver/internal/Robot.java

```diff
@@ -56,6 +57,7 @@
 import org.kaazing.k3po.driver.internal.behavior.visitor.GenerateConfigurationVisitor;
 import org.kaazing.k3po.driver.internal.netty.bootstrap.BootstrapFactory;
 import org.kaazing.k3po.driver.internal.netty.bootstrap.ClientBootstrap;
+import org.kaazing.k3po.driver.internal.netty.bootstrap.LazyClientBootstrap;
```

Comment:
Avoid treating `LazyClientBootstrap` as a special case alongside `ClientBootstrap`.
Instead, make `ClientBootstrap` lazy-capable and only use that for all client bootstrap scenarios.